src/ontology/mondo-edit.obo

```diff
@@ -206889,8 +206889,7 @@ synonym: "immunodeficiency with hyper IgM type 1" RELATED [GARD:0000073]
 synonym: "immunodeficiency with hyper-IgM, type 1" RELATED [MONDO:Lexical, OMIM:308230]
 synonym: "immunodeficiency with hyper-IgM, type 1; HIGM1" RELATED [OMIM:308230]
 synonym: "X-linked hyper IgM syndrome" RELATED [GARD:0000073]
-synonym: "X-linked hyper-IgM syndrome" EXACT [GARD:0000073]
-synonym: "X-linked hyper-IgM syndrome" EXACT [DOID:0060022]
+synonym: "X-linked hyper-IgM syndrome" EXACT [DOID:0060022, GARD:0000073]
```

Comment:
looks like you combined the omim PR with a normalization PR. nbd but would be nice to have a better system here maybe gh actions to periodically normalize axiom annotations?

Reply:
I thought this was the SOP? At least my notes said that I should run normalisation after adding the OMIMs in.. Is it better to run normalisation in a completely separate step then?